lib/modules/auth/services/auth.service.dart

```diff
@@ -14,7 +16,25 @@ class AuthService {
     return _dio;
   }
 
-// TODO - Implement API calls
+  Future<bool> login(String email, String password) {
```

Comment:
Shouldn't this return a Future<string> or something similar? Since this will basically have to return the token that will be used further by the auth provider

Reply:
True, but the method is a skeleton for now. The api call is invalid and it will need adjustments to work.

Reply:
That's also right, well then